### PR TITLE
Add contributions document

### PR DESCRIPTION
**Pull request by leobalter**
Ref gh-24

Still need to collect and add more contributions.

### CODE REVIEW

**Comment by leobalter** [comment]
@scottgonzalez @rwaldron

File: Contributions.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+- [fixed the behavior](https://lists.w3.org/Archives/Public/public-whatwg-archive/2009Oct/0057.html) of `stepUp()` and `stepDown()` for `<input type="number">` based on the behavior of the spinner widget in __jQuery UI__. (from @scottgonzalez).
+
+# ECMAScript / TC39
+
+## Current proposals:
+
+| Proposal | Champion | Stage |
+|----------|----------|-------|
+| [String padding](https://github.com/tc39/proposal-string-pad-start-end) | Jordan Harband & Rick Waldron | 3 |
```
Comment (rwaldron) [inline Contributions.md]: It's more like I originally brought forward a proposal, but Jordan really gets the credit here. Would be nice if that was clearly stated

File: Contributions.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+| [`String.prototype.at`](https://github.com/mathiasbynens/String.prototype.at) | Mathias Bynens & Rick Waldron | 0 |
+| [Function Bind Syntax](https://github.com/zenparsing/es-function-bind) | Kevin Smith | 0 |
+
+## Landed features:
+
+| Feature | Version | Champion |
+|---------|---------|----------|
+| Exponentiantion Operator | [ES2016](https://tc39.github.io/ecma262/2016/#sec-exp-operator) | [Rick Waldron](@rwaldron) |
+| [`Object.values` and `Object.entries`](https://github.com/tc39/proposal-object-values-entries) | [ES2016](https://tc39.github.io/ecma262/#sec-object.entries) | Jordan Harband (reviewed by @wycats @littledan @rwaldron) |
+
```
Comment (rwaldron) [inline Contributions.md]: So...

- `Array.from` and `Array.of`: Dave Herman & Rick Waldron
- `Array.prorotype.find` & `Array.prorotype.findIndex`: Rick Waldron
- `Object.assign`: Rick Waldron

File: Contributions.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+## Spec fixes
+
+- [TypedArrays/ArrayBuffer/DataView constructor normalization/spec reform](https://github.com/tc39/ecma262/pull/410) (@leobalter)
+
+## Test262
+
+- TODO: list contributions
+- participants from the jQuery Foundation: @rwaldron, @leobalter
+
+# [ECMAScript Internationalization API Specification (Ecma-402)](https://github.com/tc39/ecma402)
```
Comment (rwaldron) [inline Contributions.md]: - Second Edition: Rick Waldron

File: Contributions.md (hunk 2 above)
Comment (rwaldron) [inline Contributions.md]: How about "Failed Proposals"? 

- `Math.TAU` Rick Waldron

File: Contributions.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
+# Contributions
+
+This document list current and past contributions made to standards that has some relationship with the jQuery Foundation projects.
```
Comment (scottgonzalez) [inline Contributions.md]: s/list/lists/

**Comment by scottgonzalez** [comment]
We should probably be consistent with how we list people. There's currently a mix of at-name, real name, real name with link to GitHub account. I think real name with link to GitHub account is the way to go.

File: Contributions.md (hunk 1 above)
Comment (ljharb) [inline Contributions.md]: no objection to clarifying it, but imo no need either.

File: Contributions.md (hunk 2 above)
Comment (ljharb) [inline Contributions.md]: RIP `τ` ◉︵◉

Comment (leobalter) [inline Contributions.md]: I really enjoyed the discussion on it, at least: https://esdiscuss.org/topic/math-tau

**Comment by leobalter** [comment]
@scottgonzalez @rwaldron I believe fixed everything. On a follow up PR, I want to include a quick one-line introduction of the participants.

**Comment by scottgonzalez** [comment]
Looks good.

You might want to alphabetize the links at the bottom by name. This will make it easier to see if a name is already in the list when adding future contributions.

**Comment by leobalter** [comment]
Sure thing, I just did it.

File: Contributions.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+| [Function Bind Syntax](https://github.com/zenparsing/es-function-bind) | [Kevin Smith] | 0 |
+
+## Landed features:
+
+| Feature | Version | Champion |
+|---------|---------|----------|
+| `Array.from` & `Array.of` | [ES2015][Array.from] | [Dave Herman] & [Rick Waldron] |
+| `Array#find` & `Array#findIndex` | [ES2015][Array#find] | [Rick Waldron] |
+| `Object.assign` | [ES2015][Object.assign] | [Rick Waldron] |
+| Exponentiantion Operator (`**`) | [ES2016][Exp Operator] | [Rick Waldron] |
```
Comment (ljharb) [inline Contributions.md]: `s/Exponentiantion/Exponentiation`

File: Contributions.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+[Exp Operator]: https://tc39.github.io/ecma262/2016/#sec-exp-operator
+[Object.entries]: https://tc39.github.io/ecma262/#sec-object.entries
+[Array#find]: http://www.ecma-international.org/ecma-262/6.0/index.html#sec-array.prototype.find
+[Object.assign]: http://www.ecma-international.org/ecma-262/6.0/index.html#sec-object.assign
+
+## Failed proposals:
+
+| Feature | Champion |
+|---------|----------|
+| [`Math.TAU`](https://esdiscuss.org/topic/math-tau) | [Rick Waldron] |
```
Comment (markelog) [inline Contributions.md]: Do we need to mention failed attempts?

**Comment by markelog** [comment]
Can we mention @mikesherov and his involvement in estree (although, i'm not sure if that thing is still alive) and CSSWG? .

I also remembering @jaubourg was involved with xhr spec

File: Contributions.md (hunk 6 above)
Comment (scottgonzalez) [inline Contributions.md]: I think it's useful for a few reasons. It shows that we were working on standards, even if they didn't get ratified. It also makes it easier for someone else who would like to see such functionality standardized find the relevant discussions and see the reasons it wasn't approved.

Comment (leobalter) [inline Contributions.md]: the linked discussion is at least interesting and show there's not silver bullet on proposals. Some will become a feature, some will fail. Still, it's good to know we worked on it and good to learn reasons a proposal might not be approved.

**Comment by leobalter** [comment]
@markelog sure! You just pointed to information I didn't have before and this document can be improved anytime. Can we add them as a follow up PR so we land this as initial document?

@ljharb thanks! I just fixed the typo.

**Comment by mikesherov** [comment]
Thanks for the mention @markelog ;-)

File: Contributions.md (hunk 6 above)
Comment (markelog) [inline Contributions.md]: :+1:

**Comment by markelog** [comment]
> Can we add them as a follow up PR so we land this as initial document?

Totally. Estree is here - https://github.com/estree

@mikesherov, @jaubourg maybe you could provide other relevant links?

**Comment by mikesherov** [comment]
@ariya was involved heavily in ESTree as well. 

Mike Sherov

**Pull request merged**